buildchow/templates/product-template.yaml

```diff
@@ -160,5 +157,6 @@
             publish-failed-builds: false
 
     wrappers: 
-        - timeout:
-            timeout: 120
+        - raw:
+            xml: |
+               <hudson.plugins.build__timeout.BuildTimeoutWrapper><strategy class="hudson.plugins.build_timeout.impl.AbsoluteTimeOutStrategy"><timeoutMinutes>180</timeoutMinutes></strategy><operationList><hudson.plugins.build__timeout.operations.AbortOperation/></operationList></hudson.plugins.build__timeout.BuildTimeoutWrapper>
```

Comment:
any reason to not use the built-in support ? 

http://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.timeout